documentation/assemblies/upgrading/assembly-upgrade-cluster-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='assembly-upgrade-cluster-operator-{context}']
+= Upgrading the Cluster Operator
+
+[role="_abstract"]
+How you upgrade the Cluster Operator depends on the way you deployed it.
+
+Using installation files:: If you deployed the Cluster Operator using the installation YAML files, perform your upgrade by modifying the Operator installation files, as described in xref:proc-upgrading-the-co-{context}[Upgrading the Cluster Operator].
+
+Using the {OperatorHub}:: If you deployed Strimzi from {OperatorHub}, use the Operator Lifecycle Manager (OLM) to change the update channel for the Strimzi operators to a new Strimzi version.
```

Comment:
We have a note to not recommend automatic updates from the stable channel in [Deploying Strimzi from OperatorHub.io](https://strimzi.io/docs/operators/in-development/deploying.html#deploying-strimzi-from-operator-hub-str)

So I've added a note here about the automatic updates and to only use them on the version-specific channels. 

> NOTE: If you subscribe to the _stable_ channel, you can get automatic updates without changing channels.
> However, enabling automatic updates on this channel is not recommended because of the potential for missing any pre-installation upgrade steps. Use automatic upgrades only on version-specific channels.